### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by JuanLPalacios**
Implementation of desktop design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,7 +61,7 @@ <h1>My Recent Works</h1>
             <div class="grid works-list">
                 <article class="card work">
                     <div class="background">
-                        <img src="https://picsum.photos/200/300" alt="">
+                        <img src="https://picsum.photos/200/150" alt="">
                     </div>
                     <div class="content">
```
Comment (rotimiazeez) [inline index.html]: - [ ] I'd suggest you populate the work section as it is required to match the original design as much as possible😉 it also improves your page UI(User Interface)👍
- Your Implementation | Figma Design
  |

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
#### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -56,7 +61,7 @@ <h1>My Recent Works</h1>
             <div class="grid works-list">
                 <article class="card work">
```
Comment (mateo951) [inline index.html:62]: You did a great job populating your works section 👌🔥

**Review by mateo951 — APPROVED** [review]
Hi @JuanLPalacios, 

Awesome work at making the necessary adjustments requested by the previous reviewer 🙌
The design looks pretty nice 🔥🔥🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 😁

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**